django-stubs/db/models/expressions.pyi

```diff
@@ -148,30 +154,24 @@ class OuterRef(F):
     contains_aggregate: bool
     def relabeled_clone(self: Self, relabels: Any) -> Self: ...
 
-class Subquery(BaseExpression, Combinable):
+class Func(SQLiteNumericMixin, Expression):
+    function: str
+    name: str
     template: str
-    query: Query
+    arg_joiner: str
+    arity: int | None
+    source_expressions: list[Expression]
     extra: dict[Any, Any]
-    def __init__(self, queryset: Query | QuerySet, output_field: Field | None = ..., **extra: Any) -> None: ...
-
-class Exists(Subquery):
-    negated: bool
-    def __init__(self, queryset: Query | QuerySet, negated: bool = ..., **kwargs: Any) -> None: ...
-    def __invert__(self) -> Exists: ...
-
-class OrderBy(Expression):
-    template: str
-    nulls_first: bool
-    nulls_last: bool
-    descending: bool
-    expression: Expression | F | Subquery
-    def __init__(
+    def __init__(self, *expressions: Any, output_field: Field | None = ..., **extra: Any) -> None: ...
+    def as_sql(
         self,
-        expression: Expression | F | Subquery,
```

Comment:
Reordering the classes here made it **very** difficult to understand what actually changed here. I had to order them back to get a readable diff.

In the future, if you want to reorder items, please do it as a separate PR without any additional changes.